Makefile.PL

```diff
@@ -4,7 +4,9 @@ RTx     'RT-Authen-OAuth2';
 license 'gplv2';
 repository 'https://github.com/bestpractical/rt-authen-oauth2';
 
-requires_rt '4.4.0';
+# has not been well tested with versions before 4.4, but seems to work
+# fine with later 4.0.X releases.
+requires_rt '4.0.0';
```

Comment:
I um... still have two 4.0 instances.  :)  You can ask @cbrandtbuffalo about them.  But since they've already got this installed, the upstream version can require something reasonable.